### DISCUSSION TITLE
Mobox does not start on S23 Android 14

### DISCUSSION THREAD

**Discussion by RogerBrMG** [olegos2/mobox#236] [Q&A]
Whenever I try to install mobox it keeps giving me this error (both on box68 and wow64), does anyone know what it could be?
![Screenshot_20240225_121259_Termux](https://github.com/olegos2/mobox/assets/125814380/31cc9140-fefc-4412-bba9-b2bb56ee530b)

**Comment by ShinigamixKira** [comment]
Me too I thought I'm the only one hahah..reinstall Mobox and this happen..oh my god please anyone help me
